go.mod

```diff
@@ -3,18 +3,18 @@ module sigs.k8s.io/custom-metrics-apiserver
 go 1.18
```

Comment:
Let's require `go 1.19` here since it is the case for k8s 0.25.4 deps: https://github.com/kubernetes/client-go/blob/v0.25.4/go.mod